ts/webui/src/components/managementExp/ExperimentManager.tsx

```diff
@@ -251,6 +257,15 @@ class Experiment extends React.Component<{}, ExpListState> {
         }
     ];
 
+    private something = (item?: any, _index?: number, _ev?: React.FocusEvent<HTMLElement>): void => {
```

Comment:
`experimentClicked`

Reply:
Also I doubt this will get activated by tab key. No need to check or fix now though.

Reply:
had rename to `experimentClicked`